### ISSUE TITLE
Does this support CAS functionality?

### ISSUE THREAD

**Issue by zai1208** [AstusRush/AMaDiA#3]

**Comment by AstusRush** [comment]
Yes, it does. Though this question is very broad. If you want to know any specifics feel free to ask.

(Sorry for this very delayed response. I was occupied with my bachelor thesis the past few months.)
